sys-snap.pl

```diff
@@ -299,94 +301,48 @@ sub loadavg {
     return;
 }
 
-sub stop_syssnap {
-    my $pid;
-    # prevent check_status from printing to terminal
-    {
-        local *STDOUT;
-        open (STDOUT, '>', '/dev/null') or die "Can't access /dev/null";
-        $pid = &check_status();
-    }
-    if ($pid =~ /[\d+]/) {
-        #print "Test: $pid\n";
-        delete @ENV{'PATH', 'IFS', 'CDPATH', 'ENV', 'BASH_ENV'};
-        my $running_pid = "false";
-        my $ps_info = `ps -e -o pid,user,args | grep "[s]ys-snap.pl --start"`;
-        if ($ps_info =~ /^\s*([0-9]+)\s+root\s+(\/usr\/bin\/perl\s+\.\/|perl\s+)sys-snap\.pl\s+--start/ ) {
-            $running_pid = $1;
-        }
-        print "Current process: $ps_info";
-        print "Stop this process (y/n)?:";
 
-        my $choice = "0";
-        $choice = <STDIN>;
-        while ($choice !~ /[yn]/i ) {
+sub stop_syssnap {
```

Comment:
Please have stop_syssnap remove the pidfile before exit, it is currently being left behind.